### PR TITLE
refactor(app-shell): avoid using deprecated fields

### PR DESCRIPTION
**Pull request by emmenko**
A leftover from the Org-level Custom Applications release.

### CODE REVIEW

File: packages/application-shell/src/components/navbar/fetch-project-extensions-navbar.settings.graphql [hunk 1]
```diff
@@ -31,24 +31,24 @@ query FetchProjectExtensionsNavbar {
...
-            }
+        }
+        submenuLinks {
+          id
+          uriPath
+          permissions
+          defaultLabel
+          labelAllLocales {
+            locale
+            value
```
Comment (emmenko) [inline packages/application-shell/src/components/navbar/fetch-project-extensions-navbar.settings.graphql:51]: This is the main change.

File: packages/application-shell/src/components/application-shell/application-shell.spec.js [hunk 2]
```diff
@@ -911,8 +912,8 @@ describe('navbar menu links interactions', () => {
                     }),
                   }),
                 }),
-                // TODO: also test installed applications!
-                installedApplications: [],
```
Comment (emmenko) [inline packages/application-shell/src/components/application-shell/application-shell.spec.js:915]: A long overdue leftover 🙃

File: graphql-test-utils/graphql-models/custom-application-installation.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { Factory } from 'rosie';
```
Comment (emmenko) [inline graphql-test-utils/graphql-models/custom-application-installation.js:1]: FYI: as a follow up we should rewrite these test-data models to use our new setup.

File: packages/application-shell/src/components/navbar/navbar.tsx [hunk 4]
```diff
@@ -198,8 +198,8 @@ const MenuGroup = (props: MenuGroupProps) => {
     props.level === 2 && props.isActive && !props.isExpanded;
   return (
     <ul
-      id={`${props.id}-group`}
```
Comment (emmenko) [inline packages/application-shell/src/components/navbar/navbar.tsx:201]: Apparently querying for this ID (in tests) randomly fails if the value starts with a number (when using UUIDs).

image

Therefore I changed the prefix to be `group-`

Comment (CarlosCortizasCT) [reply packages/application-shell/src/components/navbar/navbar.tsx:201]: 😮

**Review by CarlosCortizasCT — APPROVED** [review]
Looks very good to me 🚀

**Review by kark — APPROVED** [review]
💯

**Pull request merged**